<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>881</width>
      <height>262</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="selCOM_CB">
           <property name="minimumSize">
            <size>
             <width>72</width>
             <height>0</height>
            </size>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::InsertAtTop</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="openCom_PB">
           <property name="minimumSize">
            <size>
             <width>77</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>OPEN PORT</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="connect_PB">
         <property name="minimumSize">
          <size>
           <width>111</width>
           <height>54</height>
          </size>
         </property>
         <property name="text">
          <string>CONNECT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="receiveData_TE">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>54</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="logo_LL">
         <property name="minimumSize">
          <size>
           <width>72</width>
           <height>54</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>72</width>
           <height>54</height>
          </size>
         </property>
         <property name="text">
          <string>LOGO</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>IP v4 ADDRESS</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QPushButton" name="writePortUDP_PB">
           <property name="text">
            <string>WRITE</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="readSWver_LE">
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Software Version</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="readPortUDP_PB">
           <property name="text">
            <string>READ</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="readPortTCP_LE"/>
         </item>
         <item row="4" column="3">
          <widget class="QLineEdit" name="writeAPN_LE">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="writePortUDP_LE">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Mode TCP/UDP</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QPushButton" name="readSWver_PB">
           <property name="text">
            <string>READ</string>
           </property>
          </widget>
         </item>
         <item row="4" column="4">
          <widget class="QPushButton" name="writeAPN_PB">
           <property name="text">
            <string>WRITE</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="readPortUDP_LE"/>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>APN</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="readAdrIPv4_PB">
           <property name="text">
            <string>READ</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="readAPN_PB">
           <property name="text">
            <string>READ</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="readPortTCP_PB">
           <property name="text">
            <string>READ</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="writePortTCP_PB">
           <property name="text">
            <string>WRITE</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>PORT TCP</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>PORT UDP</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLineEdit" name="writeAdrIPv4_1_LE">
             <property name="maximumSize">
              <size>
               <width>33</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="writeAdrIPv4_2_LE">
             <property name="maximumSize">
              <size>
               <width>33</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="writeAdrIPv4_3_LE">
             <property name="maximumSize">
              <size>
               <width>33</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="writeAdrIPv4_4_LE">
             <property name="maximumSize">
              <size>
               <width>33</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="readAPN_LE">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QPushButton" name="writeAdrIPv4_PB">
           <property name="text">
            <string>WRITE</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="writeTCPUDPMode_CB">
             <property name="insertPolicy">
              <enum>QComboBox::InsertAlphabetically</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="writePortTCP_LE">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="readAdrIPv4_LE"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="readTCPUDPMode_LE"/>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="readTCPUDPMode_PB">
           <property name="text">
            <string>READ</string>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QPushButton" name="writeTCPUDPMode_PB">
           <property name="text">
            <string>WRITE</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
